.grunt-config/webpack.packages.js

```diff
@@ -87,3 +89,53 @@ module.exports = {
 	dev: devConfig,
 	prod: prodConfig,
 };
+
+function getPackagesBasedOnProjectDeps() {
+	const { dependencies } = require( '../package.json' );
+
+	return Object.keys( dependencies )
+		.filter( ( packageName ) => packageName.startsWith( '@elementor/' ) )
+		.map( ( packageName ) => {
+			const pkgJSON = fs.readFileSync( path.resolve( __dirname, `../node_modules/${packageName}/package.json` ) );
+
+			const { main, module } = JSON.parse( pkgJSON );
+
+			return {
+				mainFile: module || main,
+				packageName,
+			}
+		} )
+		.filter( ( { mainFile } ) => !! mainFile )
+		.map( ( { mainFile, packageName } ) => ( {
+			name: packageName.replace( '@elementor/', '' ),
+			path: path.resolve( __dirname, `../node_modules/${ packageName }`, mainFile ),
+		} ) );
+}
+
+function getPackagesBasedOnLocalRepo() {
+	const repoPath = process.env.ELEMENTOR_PACKAGES_PATH;
+	const relevantDirs = [ 'packages/core', 'packages/libs' ]
+
+	if ( ! repoPath && fs.existsSync( repoPath ) ) {
```

Comment:
```suggestion
	if ( ! repoPath || ! fs.existsSync( repoPath ) ) {
```

I would also consider putting it in a different condition and different error message, I think it's much better DX

Reply:
Done

Reply:
My bad